pkg/network/additional_networks.go

```diff
@@ -10,12 +11,12 @@ import (
 	uns "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
-// renderAdditionalNetworksCRD returns the manifests of the NetworkAttachmentDefinition.
-func renderAdditionalNetworksCRD(manifestDir string) ([]*uns.Unstructured, error) {
+// renderRawNetworkCRD returns the manifests of the NetworkAttachmentDefinition.
+func renderRawNetworkCRD(manifestDir string) ([]*uns.Unstructured, error) {
```

Comment:
I don't think this should change. Instead, we should always render the CRD whenever multus is enabled. There should only be one instance of the CRD in `bindata/`. So, I would set up the directory structure like this:

```
/bindata/additional-networks/crd/000-network-attachment-crd.yaml
/bindata/additional-networks/raw/network-attachment.yaml
/bindata/additional-networks/sriov/network-attachment.yaml (plus the other sr-iov files)
```

Reply:
Done